src/ServerMain.cpp

```diff
@@ -129,6 +129,10 @@ int main(int argc, char** argv) {
       optionFactory.getProgramOption<"request-body-limit">(),
       "Set the maximum size for the body of requests the server will process. "
       "Set to zero to disable the limit.");
+  add("cache-service", optionFactory.getProgramOption<"cache-service">(),
```

Comment:
I would suggest the name
`cache-service-clauses` (for the cli as well as for the runtime parameter)
And document that this also disables the sibling optimization (if it does).
@hannahbast What name would you suggest?